contracts/translation/translation.go

```diff
@@ -0,0 +1,24 @@
+package translation
+
+//go:generate mockery --name=Translation
+type Translation interface {
+	// Load translations from path.
+	Load(path string, locale ...string) error
+	GetDefaultLocale() string
+	//
+	Language(locale string) Language
+	// Add translations to application.
+	Add(name, locale string, val any)
```

Comment:
S: Consistent with Laravel: https://github.com/laravel/framework/blob/aef89589ea70e0081c139b06550220cc75f20ea6/src/Illuminate/Translation/Translator.php#L256
```suggestion
	AddLines(lines map[string]any, locale string)
```